### PR TITLE
This is a title

### PR DESCRIPTION
**Pull request by CodelingoBotTester**
This is a body

### CODE REVIEW

File: example.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ func main() {
 	fmt.Println("Hello")
 }
 
+// The original line
```
Comment (CodelingoBotTester) [inline example.go:9]:
```diff
- // The original line
+ // Some {{ .commentVar }}
```

Change `commentVar` by commenting: `@CodeLingoBot set commentVar "your comment"`

Comment (Daanikus) [reply example.go:9]: @CodelingoBotTester set commentVar "the ultimate test"

Comment (CodelingoBotTester) [reply example.go:9]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply example.go:9]: @CodeLingoBot set funcComment "your comment"

Comment (Daanikus) [reply example.go:9]: @CodelingoBotTester set commentVar "the ultimate test"

Comment (CodelingoBotTester) [reply example.go:9]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (CodelingoBotTester) [reply example.go:9]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

File: example.go [hunk 2]
```diff
@@ -6,6 +6,7 @@ func main() {
...
 func NoComment() {
 	fmt.Println("Exported but no comment")
 }
```
Comment (CodelingoBotTester) [inline example.go]:
```diff
- // The original line
+ // Some {{ .funcComment }}
```

Change `commentVar` by commenting: `@CodeLingoBot set funcComment "your comment"`

Comment (Daanikus) [reply example.go]: @CodelingoBotTester set funcComment "your comment"

Comment (CodelingoBotTester) [reply example.go]: unexpected command 

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply example.go]: @CodelingoBotTester rewrite set funcComment "your comment"

Comment (CodelingoBotTester) [reply example.go]: error handling failed for:POST https://api.github.com/repos/Daanikus/ir-pr-test/pulls/1/comments: 422 Validation Failed [{Resource:PullRequestReviewComment Field:pull_request_review_id Code:custom Message:pull_request_review_id must be pending}]: POST https://api.github.com/repos/Daanikus/ir-pr-test/pulls/1/comments: 422 Validation Failed [{Resource:PullRequestReviewComment Field:pull_request_review_id Code:custom Message:pull_request_review_id must be pending}]

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

File: example.go [hunk 3]
```diff
@@ -4,8 +4,9 @@ import "fmt"
 
 func main() {
```
Comment (CodelingoBotTester) [inline example.go]:
```diff
- // The original line
+ // Some {{ .sometingElse }}
```

Change `commentVar` by commenting: `@CodeLingoBot rewrite set somethingElse "your comment"`

Comment (Daanikus) [reply example.go]: @CodelingoBotTester rewrite set sometingElse "your comment"

Comment (CodelingoBotTester) [reply example.go]: error handling failed for:POST https://api.github.com/repos/Daanikus/ir-pr-test/pulls/1/comments: 422 Validation Failed [{Resource:PullRequestReviewComment Field:pull_request_review_id Code:custom Message:pull_request_review_id must be pending}]: POST https://api.github.com/repos/Daanikus/ir-pr-test/pulls/1/comments: 422 Validation Failed [{Resource:PullRequestReviewComment Field:pull_request_review_id Code:custom Message:pull_request_review_id must be pending}]

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (CodelingoBotTester) [reply example.go]: Cannot perform rewrite on an outdated comment

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (CodelingoBotTester) [reply example.go]: Cannot perform rewrite on an outdated comment

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (CodelingoBotTester) [reply example.go]: Cannot perform rewrite on an outdated comment

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply example.go]: Blah blah blah

Comment (CodelingoBotTester) [reply example.go]: Cannot perform rewrite on an outdated comment

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (CodelingoBotTester) [reply example.go]: Cannot perform rewrite on an outdated comment

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply example.go]: yoyyoyoyo

Comment (CodelingoBotTester) [reply example.go]: Cannot perform rewrite on an outdated comment

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (CodelingoBotTester) [reply example.go]: Cannot perform rewrite on an outdated comment

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply example.go]: yoyoyoy

Comment (CodelingoBotTester) [reply example.go]: Cannot perform rewrite on an outdated comment

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (CodelingoBotTester) [reply example.go]: Cannot perform rewrite on an outdated comment

---
<details>
<summary>CodeLingoBot commands and options</summary>

You can trigger CodeLingo actions by commenting on this PR:

+ `@CodelingoBotTester  review` reviews the PR based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  review`</summary>
    + `@CodelingoBotTester  review ignore [reason]` ignores the issue for current and future reviews with an optional reason why.
    + `@CodelingoBotTester  review un-ignore` un-ignores the issue for current and future reviews.
+ `@CodelingoBotTester  rewrite` makes a PR to this PR fixing the issues found based on default Tenets for the repo's language or custom Tenets added in a codelingo.yaml file in the root of the repo. <details><summary>You can trigger CodeLingo actions by replying to comments generated by`@CodelingoBotTester  rewrite`</summary>

    + `@CodelingoBotTester  rewrite set <varname>` sets the value of a variable in a rewrite template.

Mention @CodelingoBotTester  to reach us.

</details>

Comment (Daanikus) [reply example.go]: yo

**Pull request closed without merging**